readthedocs/misc/v2-migration-guide.rst

```diff
@@ -202,6 +202,12 @@ is decided by Telegram.
 The ``telethon.errors`` module continues to provide custom errors used by the library such as
 ``TypeNotFoundError``.
 
+Errors are all now immutable - you can't modify them. This means the library will not correct
+weird errors such as FLOOD_WAIT_0 (which was previously modified to FLOOD_WAIT_1). When the
```

Comment:
The fact the errors are immutable does not mean the library won't correct weird errors, so I think that remark could be changed with something like "In addition, the library will not patch errors" or similar.